scanner/matcher/updater/vuln/updater_test.go

```diff
@@ -46,19 +47,22 @@ func (t *testLocker) Lock(ctx context.Context, s string) (context.Context, conte
 	return t.locker.Lock(ctx, s)
 }
 
-func testHTTPServer(t *testing.T) (*httptest.Server, time.Time) {
+func testHTTPServer(t *testing.T, content func(r *http.Request) io.ReadSeeker) (*httptest.Server, time.Time) {
 	now, err := http.ParseTime(time.Now().UTC().Format(http.TimeFormat))
 	require.NoError(t, err)
-	buf := strings.NewReader("test")
 	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		http.ServeContent(w, r, "test-file", now, buf)
+		http.ServeContent(w, r, "test-file", now, content(r))
 	}))
 	t.Cleanup(srv.Close)
 	return srv, now
 }
 
 func TestSingleBundleUpdate(t *testing.T) {
-	srv, now := testHTTPServer(t)
+	t.Setenv("ROX_SCANNER_V4_MULTI_BUNDLE", "false")
```

Comment:
nit: consider using `testutils.MustUpdateFeature(t, features.ScannerV4MultiBundle, false)` here and the equiv in `TestMultiBundleUpdate` if it fits the use case.